drivers/hmis/spec/factories/hmis/data_sources.rb

```diff
@@ -6,8 +6,8 @@
 
 FactoryBot.define do
   factory :hmis_data_source, class: 'GrdaWarehouse::DataSource' do
-    sequence(:id, 1)
-    authoritative { true }
+    sequence(:id) { |n| n + 1 }
+    sequence(:authoritative, &:zero?)
```

Comment:
what's this doing?

Reply:
The linter made me do that last line. If it makes more than one, only the first one will be authoritative.

Reply:
Hmm, ok.